homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/mathoperation/MathOperationsDemo.java

```diff
@@ -0,0 +1,32 @@
+package com.tutrit.java.quickstart.homework.mathoperation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MathOperationsDemo {
+
+    Logger log = LoggerFactory.getLogger("MathOperations ");
+
+    int sum(int a, int b) {
+        log.info("Sum {}", a + b);
+        return a + b;
+
+    }
+
+    int subtraction(int a, int b) {
+        log.info("Subtraction {}", a - b);
+        return a - b;
+    }
+
+    float division(int a, int b) {
+        log.info("Division {}", a / b);
+        return (float) a / b;
```

Comment:
maybe double works better? 
also, it's less error prone if you log and return the same var, like 
```suggestion
    float division(int a, int b) {
        float result = (float) a / b
        log.info("Division {}", result);
        return result;
```
Also, in this case you don't need to perform division twice.

Reply:
fixed